swssconfig/sample/00-copp.config.json

```diff
@@ -55,5 +55,19 @@
             "red_action":"drop"
         },
         "OP": "SET"
+    },
```

Comment:
Since this change is not related to the problem that is being solved in this PR. Can you move this change to a separate PR

Reply:
Yes, moved this change to new PR https://github.com/Azure/sonic-swss/pull/1300 .